docs/content/en/install/doks/_index.md

```diff
@@ -0,0 +1,332 @@
+---
+title: "DigitalOcean Kubernetes"
+linkTitle: "DigitalOcean DOKS"
+weight: 2
+description: >
+  Complete **production** ready environment of K8ssandra on DigitalOcean Kubernetes (DOKS).
+---
+
+[DigitalOcean Kubernetes](https://www.digitalocean.com/products/kubernetes/) or "DOKS" is a managed Kubernetes environment on [DigitalOcean](https://www.digitalocean.com/) (DO). DOKS is a fully managed experience; it handles the management/upgrading of the Kubernetes cluster master as well as autoscaling of "nodes" through "node pools."
+
+{{% alert title="Tip" color="success" %}}
+Also available in followup topics are post-install steps and role-based considerations for [developers]({{< relref "/quickstarts/developer">}}) or [Site Reliability Engineers]({{< relref "/quickstarts/site-reliability-engineer">}}) (SREs).
+{{% /alert %}}
+
+## Deployment
+
+This topic covers provisioning the following infrastructure resources.
+
+* 1x VPC Network
+* 1x DOKS cluster
+* 2x Node Pools
+  * 3x Cassandra Kubernetes workers
+    * 32 vCPUs
```

Comment:
these seem like pretty large instances. I was not able to create these with my starter DO account without putting in a request. Should we note this and possibly provide a more conservative set of values for a "dev" deployment?

Reply:
Let's revisit instance sizing / environments in a separate PR based on our conversation yesterday.